_data/accessibility-tests/list.yml

```diff
@@ -0,0 +1,55 @@
+title: List
+component_status: pass
+test_items:
+# Zoom/screen magnification items
+  - summary: Text retains functionality when screen is magnified.
+    summary_additional: |
+      When you zoom into 200%,
+      you can still see all list content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+# Screen reader items
+  - summary: Screen reader announces list.
+    summary_additional: |
+      When you listen to a list with a screen reader,
+      the screen reader announces that it is a list and tells you how many items are in the list.
+      For example, "List; four items."
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.1
```

Comment:
**issue:** The "retested" date on this item is listed as 05/02/2024, which means it should be tagged as 3.8.0. 

**note:** The process doc was not clear about including "retested" dates here, so I updated the instructions for finding the `version_tested` value. 

```suggestion
    version_tested: 3.8.0
```

Reply:
updated version tested value to 3.8.0

Reply:
Resolved in [f359dbc](https://github.com/uswds/uswds-site/pull/2746/commits/f359dbc93ef16ed7a18a20dd41c5320ebc2bec63)